zone/client.cpp

```diff
@@ -3761,7 +3761,93 @@ void Client::Escape()
 	MessageString(Chat::Skills, ESCAPE);
 }
 
-float Client::CalcPriceMod(Mob* other, bool reverse)
+float Client::CalcClassicPriceMod(Mob* other, bool reverse) {
+	float priceMult = 0.8f;
+
+	if (other && other->IsNPC())
+		{
+		int factionlvl = GetFactionLevel(CharacterID(), other->CastToNPC()->GetNPCTypeID(), GetRace(), GetClass(), GetDeity(), other->CastToNPC()->GetPrimaryFaction(), other);
+		int32 cha = GetCHA();
+
+		if (factionlvl <= FACTION_AMIABLY) {
+			cha += 11;		// amiable faction grants a defacto 11 charisma bonus
+		}
+
+		uint8 greed = other->CastToNPC()->GetGreedPercent();
+
+		// Sony's precise algorithm is unknown, but this produces output that is virtually identical
+		if (factionlvl <= FACTION_INDIFFERENTLY)
```

Comment:
Same line brackets, e.g. `if (condition) {` and `} elseif (condition) {` and `} else {`

Reply:
Fixed.